vnext/jest.config.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  resolver: './jest-resolver.js',
```

Comment:
Should move these to `src/jest` to merge with stock utils and avoid any more root-level pollution.

Reply:
I don't agree with my previous opinion.

Having a common pattern among packages like "jest-resolver.js" seems better than special casing to RN stock.